engine/class_modules/sc_death_knight.cpp

```diff
@@ -5788,6 +5830,16 @@ struct heart_strike_t : public death_knight_melee_attack_t
   int n_targets() const override
   { return p() -> in_death_and_decay() ? aoe + as<int>( p() -> spec.death_and_decay_2 -> effectN( 1 ).base_value() ) : aoe; }
 
+  double composite_da_multiplier( const action_state_t* state ) const override
+  {
+    double m = death_knight_melee_attack_t::composite_da_multiplier( state );
+    if ( p() -> conduits.withering_plague -> ok() && td(state -> target) -> dot.blood_plague -> is_ticking() )
```

Comment:
Same thing here, there should be a function composite_target_da_multiplier (or similar) which you can use.

The usgae of target data looks much better than at the other place.